app/src/main/java/de/westnordost/streetcomplete/quests/note_discussion/AttachPhotoFragment.kt

```diff
@@ -42,6 +41,8 @@ class AttachPhotoFragment : Fragment() {
 
     private lateinit var noteImageAdapter: NoteImageAdapter
 
+    private val takePhoto = registerForActivityResult(ActivityResultContracts.TakePicture(), ::onTakePhotoResult)
```

Comment:
You tested that, right? I wonder if/how this is possible, because on creation, the fragment is not attached to any Activity yet, but you need the activity to register an activity result contract to, no?

Reply:
Yeah ok, seems to work like that

Reply:
Yes, I tested the change. I even enabled the "don't keep activities" developer option.